recipe/meta.yaml

```diff
@@ -8,29 +8,44 @@ source:
     fn: flann-{{ version }}.tar.gz
     url: https://github.com/mariusmuja/flann/archive/{{ version }}.tar.gz
     sha256: b23b5f4e71139faa3bcb39e6bbcc76967fbaf308c4ee9d4f5bfbeceaa76cc5d3
+    patches:
+        - cmake.patch  # [linux]
 
 build:
-    number: 0
-    skip: true    # [win]
+    number: 1
+    features:
+        - vc9   # [win and py27]
+        - vc14  # [win and py>=35]
 
 requirements:
     build:
         - gcc     # [unix]
```

Comment:
Please add the following...

```yaml
        - {{ compiler('cxx') }}     # [win]
```

Once added all `vc` features and `vc` dependencies can be dropped.